packages/gateway/src/features/location/root-resolvers.ts

```diff
@@ -10,26 +10,15 @@
  */
 
 import { GQLResolver } from '@gateway/graphql/schema'
-import { fetchFHIR } from '@gateway/features/fhir/service'
+import { fetchLocationChildren } from '@gateway/location'
+import { UUID } from '@opencrvs/commons'
 
 export const resolvers: GQLResolver = {
   Query: {
-    async locationsByParent(_, { parentId }, { headers: authHeader }) {
-      const bundle = await fetchFHIR(`/Location?partof=${parentId}`, authHeader)
-      return bundle.entry.map((entry: { resource: {} }) => entry.resource)
-    },
```

Comment:
I couldn't find where this & locationById is used in the client so decided to remove them.

Reply:
Mark these to CHANGELOG as breaking changes please